jobbergate-api/README.rst

```diff
@@ -1,3 +1,47 @@
+.. raw:: html
+
+   <p align="center">
+     <img
+       src="https://img.shields.io/github/workflow/status/omnivector-solutions/jobbergate/Test/main?label=main-build&logo=github&style=plastic"
+       alt="main build"
+     />
+     <img
+       src="https://img.shields.io/github/issues/omnivector-solutions/jobbergate?label=issues&logo=github&style=plastic"
```

Comment:
I think the issues and prs badges should be left at the top level, not on the subproject.

Reply:
solved in [2475c7c](https://github.com/omnivector-solutions/jobbergate/pull/155/commits/2475c7cb2f61a1e952c2e8ef24e32f0f2eb6dc1a)